<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clone</class>
 <widget class="QDialog" name="Clone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GitUI Clone</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelOperacja">
     <property name="font">
      <font>
       <family>Nimbus Roman No9 L</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="6">
    <widget class="QTreeView" name="wyswietlaczDrzewiak"/>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QTextBrowser" name="wyswietlacz"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelURL">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditURL"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_2Nazwa"/>
   </item>
   <item row="3" column="2">
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_2Nazwa">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="pushButton_2Back">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
